### PR TITLE
Create WINGS_TO_FLY

### PR DESCRIPTION
**Pull request by Rathore24**
Haiku prepared by me.

### CODE REVIEW

File: _haikus/WINGS_TO_FLY [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: WINGS_TO_FLY
+author: RAYNORX
+---
+
+I dreamt to meet<br>
```
Comment (ltagliaferri) [inline _haikus/WINGS_TO_FLY]: This line needs one more syllable, you can try:
```
I had dreamt to meet<br>
```

File: _haikus/WINGS_TO_FLY [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+up with Sky,but found myself<br>
+without Wings to Fly high.<br>
```
Comment (ltagliaferri) [inline _haikus/WINGS_TO_FLY]: This line is one syllable too many, you can try:
```
without Wings to Fly.<br>
```

File: _haikus/WINGS_TO_FLY (hunk 1 above)
Comment (Rathore24) [reply _haikus/WINGS_TO_FLY]: But 5 are already complete
I dre amt to me et
I suppose

**Comment by Rathore24** [comment]
I suppose this is it

**Comment by ltagliaferri** [comment]
Closing this as requested changes have not been addressed.

**Comment by Rathore24** [comment]
Sir does that works.
Actually I was not having laptop at that time to make changes to haiku
created by me on your suggestions.
Today I would post this one as pr if it is possible.

**Pull request closed without merging**